iOS-Email-Client/Controllers/Login/SignUpViewController.swift

```diff
@@ -284,3 +285,15 @@ class SignUpViewController: UIViewController{
         }
     }
 }
+
+extension SignUpViewController: UITextFieldDelegate {
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        guard !string.isEmpty else {
+            return true
+        }
+        guard range.location > 0 else {
+            return string.range(of: "[a-z]", options: .regularExpression) != nil
+        }
+        return string.range(of: "^[.a-z0-9_-]*$", options: .regularExpression) != nil
```

Comment:
This regex is not right. I'll wait this fix in the next PR

Reply:
why not now?

Reply:
already have it in another branch